chapters/3-measurement.tex

```diff
@@ -37,125 +39,210 @@
 
 %-----------------------------------------------------------------------------------------------
 
-\section{Translating research design to master data}
+\section{Translating research design to a data plan}
 
 In most projects, more than one data source is needed to answer the research question.
 These could be multiple survey rounds,
 data acquired from different  partners (e.g. administrative data,
 web scraping, implementation monitoring, etc)
 or complex combinations of these.
-For example, you may have different \textbf{units of observation}\sidenote{
-	The \textbf{unit of observation} is the unit at or for which data is collected. See
-	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}},
-and their level may vary from round to round.
-
 However your study is structured, you need to know how to link data from all sources
 and analyze the relationships between the units that appear in them
 to answer all your research questions.
 You might think that you are able to keep all the relevant details in your head,
 but your whole research team is unlikely to have the same understanding,
 at all times, of all the datasets required.
 The only way to make sure that the full team shares the same understanding
-is to create \textit{master datasets} and a \textit{data map}.
-
-\index{Master datasets}\textbf{Master datasets} serve three key functions.
-First, they list all the units that are eligible for the study,
-and enable you to map the data to the research design.
-Second, in designs where your team has direct control over interventions or other field work,
-they allow you to plan sampling and treatment assignment before going to the field.
-Finally, they constitute the single unambiguous location where all information 
-related to the implementation and validity of your research is stored,
-as well as all information needed to correctly identify any observation in any of your project's datasets.
-The \textbf{data map} diagrams each data source the project will use, 
-the unit of response, frequency of measurement, 
-and the level(s) at which it can be linked to other datasets within the framework. 
-
-
-\subsection{Creating master datasets and a data map}
-
-A \textbf{master dataset}\sidenote{
+is to create a \textbf{data plan}\index{Data plan}.\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Plan}}
+DIME's data plan template has three components; 
+one \textit{data linkage table},\index{Data linkage table}
+one or several \textit{master datasets}\index{Master datasets}
+and one or several \textit{data flow charts}.\index{Data flowchart} 
```

Comment:
flow chart one word or two?

Reply:
Agreed `flowchart` @kbjarkefur

Reply:
added to https://github.com/worldbank/dime-data-handbook/issues/476